wl/utils/Utils.java

```diff
@@ -21,18 +21,255 @@
 
 import java.io.*;
 import java.nio.file.*;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
 import java.util.*;
 
+/**
+ * Miscellaneous utility functions.
+ */
 public abstract class Utils {
-	public static final TreeMap<File, TreeMap<String, Value>> _staticprofiles = new TreeMap<>();
+	private static class ChecksummedFile implements Comparable {
+		public final File file;
+		public final String checksum;
+
+		public ChecksummedFile(File f) {
+			file = f;
+			checksum = cs();
+		}
+
+		private String cs() { return file.isFile() ? checksum(file) : ""; }
+
+		public boolean valid() { return cs().equals(checksum); }
```

Comment:
Rename the String `checksum`  to prevent confusion with the function `checksum(file)`

Reply:
done